.gitignore

```diff
@@ -29,3 +29,4 @@ __pycache__
 data/generated.json
 data/tags.json
 content/language/*.md
+pnpm-lock.yaml
```

Comment:
We use `bun` on this repository. Aside, lock files should not be ignored. I'd recommend removing this.